wasted-app/RestaurantFoodControl.cs

```diff
@@ -14,29 +15,70 @@ public partial class RestaurantFoodControl : UserControl
         private readonly ServicesController services = ServicesController.Instance;
         public RestaurantFoodControl(Restaurant restaurant)
         {
-            InitializeComponent();
             LoggedRestaurant = restaurant;
+            InitializeComponent();
+            InitializeComboBoxItems();
             GetRestaurantFoodItems();
-            ListRestaurantFoodItems();
+            ListRestaurantFoodItems(Foods);
         }
 
         private void GetRestaurantFoodItems()
         {
             Foods = FoodUtilities.GetFoodByRestaurantId(LoggedRestaurant.Id);
         }
 
-        private void ListRestaurantFoodItems()
+        private void ListRestaurantFoodItems(IEnumerable <Food> foods)
         {
+            foodPanel.Controls.Clear();
             foreach (var food in Foods)
             {
-                var foodItem = new FoodControl(food.Name, FoodUtilities.GetFoodTypeName(food.IdTypeOfFood), food.Price.ToString("0.00"));
+                var foodItem = new FoodControl(food.Name, FoodUtilities.GetFoodTypeNameById(food.IdTypeOfFood), food.Price.ToString("0.00"));
                 foodPanel.Controls.Add(foodItem);
             }
         }
 
+        private void InitializeComboBoxItems()
+        {
+            var foodTypesNames = FoodUtilities.GetAllFoodTypesNames();
+            foreach (var foodTypeName in foodTypesNames)
+            {
+                typeComboBox.Items.Add(foodTypeName);
+            }
+        }
+
         private void BackButton_Click(object sender, EventArgs e)
         {
             MainForm.mainForm.panel.Controls.Remove(this);
         }
+
+        private void AddFoodButton_Click(object sender, EventArgs e)
+        {
+            addFoodPanel.Visible = true;
+        }
+
+        private void AddFoodConfirmButton_Click(object sender, EventArgs e)
+        {
+            var id = FoodUtilities.GetFirstAvailableFoodId();
+            var dateTime = DateTime.Now;
+            var restaurantId = LoggedRestaurant.Id;
+            var name = nameInput.Text;
+            var price = decimal.Parse(priceInput.Text);
+            var type = FoodUtilities.GetFoodTypeIdByName(typeComboBox.SelectedItem.ToString());
```

Comment:
I don't know if there's a better way to get food type id, other than search the whole database again?